spotifycli/test/test_spotifycli.py

```diff
@@ -0,0 +1,163 @@
+from io import StringIO
+from pathlib import Path
+from subprocess import PIPE
+from unittest import skip
+from unittest.mock import patch
+
+import dbus
+import dbusmock
+
+from spotifycli.spotifycli import main
+from spotifycli.version import __version__
+
+
+class TestSpotifyCLI(dbusmock.DBusTestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.start_session_bus()
+        cls.dbus_con = cls.get_dbus(system_bus=False)
```

Comment:
Each time we start running unit tests, this `classmethod` sets up the dbus connection.  Could probably get away with putting this inside the plain `setUp` that is called for each test, but I think this is more efficient and works fine.